src/test/java/org/opensrp/web/rest/it/SearchResourceTest.java

```diff
@@ -226,14 +234,15 @@ private Client createOneSearchableClient() {
 		otherClient.withIdentifier("fsdf", "sfdf");
 		otherClient.withAttribute("sfdf", "sfdf");
 		otherClient.withAttribute("alt_phone_number",phoneNumber);
+		otherClient.withAttribute("phone_number",phoneNumber);
```

Comment:
Name the PR correctly

Reply:
It will conflict with the one pointed to master

Reply:
Eg. **Enable search by alternate phone number attribute and fix search by phone number** and then you can add the suffix (v2.2) on the title of this PR while leaving the master one as-is.